src/drivers/mct/cime_config/namelist_definition_drv.xml

```diff
@@ -4266,4 +4266,17 @@
     </values>
   </entry>
 
+  <entry id="seq_flux_atmocn_minwind">
+    <type>real</type>
+    <category>seq_flux_mct</category>
+    <group>seq_flux_mct_inparm</group>
+    <desc>
+       minimum wind speed for atmOcn flux calculations
```

Comment:
Can you change this to say "...idealized flux calculations" here so it's clear what this is for?

Reply:
oh wait, is this always operating even in normal simulations?